### PR TITLE
[fixed] Modal.removePortal not called when using closetimeoutMS

### PR DESCRIPTION
**Pull request by seanich**
Acceptance Checklist:
- [x] All commits have been squashed to one.
- [x] The commit message follows the guidelines in `CONTRIBUTING.md`.
- [x] Documentation (README.md) and examples have been updated as needed.
- [x] If this is a code change, a spec testing the functionality has been added.
- [x] If the commit message has [changed] or [removed], there is an upgrade path above.

### CODE REVIEW

File: src/components/Modal.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ export default class Modal extends Component {
         this.portal.closeWithTimeout();
       }
 
-      setTimeout(() => this.removePortal, closesAt - now);
+      setTimeout(() => this.removePortal(), closesAt - now);
```
Comment (diasbruno) [inline src/components/Modal.js]: That is correct. You can remove the arrow function and it should work.

`setTimeout(this.removePortal, closesAt - now);`

Comment (seanich) [inline src/components/Modal.js]: Good point, I've updated my commit to remove the arrow function.

**Comment by diasbruno** [comment]
Thank you, @seanich!

**Comment by diasbruno** [comment]
Release `v2.2.1`.

**Pull request merged**